integrationtests/src/test/java/org/neo4j/storeupgrade/StoreUpgradeIntegrationTest.java

```diff
@@ -265,10 +265,10 @@ public void migratingOlderDataAndThanStartAClusterUsingTheNewerDataShouldWork()
         @Parameterized.Parameters( name = "{0}" )
         public static Collection<String[]> parameters()
         {
-            return Arrays.<String[]>asList(
+            return Arrays.asList(
```

Comment:
Yes, but why? Didn't we just remove support for migrating them? So why are they still a part of this test? :)